_health-care/_js/_form.js

```diff
@@ -1250,10 +1297,14 @@ var HealthApp = (function() {
     }
 
     // Set up auto-save on any changes to the form.
-    getFormRoot().addEventListener("change",
+    formRoot.addEventListener("change",
         function (e) {
           saveForm(formRoot, e);
         });
+
+    $(allInputs).on('blur', function() {
+      validate(this);
```

Comment:
I thought that `change` would be triggered too often, but after reading the docs again and testing the 2 I think `change` actually makes more sense here than `blur` (didn't realize that `change` triggers when something is selected for selects, checkboxes, and radio buttons, but on a text input triggers when it loses focus, which is why I was using `blur` in the first place). Good to know1